src/prefabs/creature.js

```diff
@@ -58,9 +59,10 @@ class Creature extends Phaser.Sprite {
 
     update() {
         this.hpText.text = 'HP: ' + this.hp_curr + '/' + this.hp_max;
+        this.game.physics.arcade.collide(this, this.game.world.getByName('Platforms'));
         this.game.physics.arcade.collide(this, this.game.world.getByName('Players'));
         this.game.physics.arcade.collide(this, this.game.world.getByName('Enemies'));
-        this.game.physics.arcade.collide(this, this.game.world.getByName('Platforms'));
+        if(this.busy) this.stop();
```

Comment:
OK, will use {} like You asked.
Yes, just reordering was sufficient to make it work. Probably when the collision of the player with the enemy was called it pushed the creature through the platforms and then the collision with platforms didn't do anything.